crossfire/__init__.py

```diff
@@ -57,12 +58,11 @@ def occurrences(
     )
 
 
-def _make_flatten(data, keys):
-    for item in data:
-        for key in keys:
-            item.update({f"{key}_{k}": v for k, v in item.get(key).items()})
-            item.pop(key)
-    return data
+def flatten_df(row, column_name):
+    column_data = row[column_name]
+    return Series(
+        {f"{column_name}_{key}": value for key, value in column_data.items()}
+    )
```

Comment:
I had to use dict here to set the column name as we are doing with dict `contextInfo_context1`...